tools/presto/presto_fixture.ts

```diff
@@ -0,0 +1,68 @@
+import * as presto from "presto-client";
+
+import { execSync } from "child_process";
+import * as dbadapters from "df/api/dbadapters";
+import { prestoExecute } from "df/api/dbadapters/presto";
+import { sleep, sleepUntil } from "df/common/promises";
+import { IHookHandler } from "df/testing";
+
+const USE_CLOUD_BUILD_NETWORK = !!process.env.USE_CLOUD_BUILD_NETWORK;
+const DOCKER_CONTAINER_NAME = "presto-df-integration-testing";
+// We remap this to something more unique.
+const PRESTO_SERVE_PORT = 8080;
+
+export const prestoTestCredentials = {
+  host: "127.0.0.1",
+  port: 1234,
+  user: "df-integration-tests",
+  catalog: "memory",
+  schema: "default"
+};
+
+export class PrestoFixture {
```

Comment:
looks like a lot of this can be common-ified into a `DockerFixture`?

Reply:
If there comes a third docker image then will do!